alpine/APKBUILD.in

```diff
@@ -10,7 +10,7 @@ depends="json-c c-ares iproute2 python3 bash"
 makedepends="ncurses-dev net-snmp-dev gawk texinfo perl
     acct autoconf automake bash binutils bison bsd-compat-headers build-base
     c-ares c-ares-dev ca-certificates cryptsetup-libs curl device-mapper-libs
-    expat fakeroot flex fortify-headers gdbm git gmp isl json-c-dev kmod
+    expat fakeroot flex fortify-headers gdbm git gmp isl-dev json-c-dev kmod
```

Comment:
FRR does not use isl.  Why not remove it from the build dependancies of alpine instead?

Reply:
Sure. I'll remove that.

Reply:
It's done.